### PR TITLE
Add jest and 'test:jest' script

### PR DESCRIPTION
**Pull request by paninaro**
Deleted old/broken test files

### CODE REVIEW

File: packages/gui/package.json [hunk 1]
```diff
@@ -106,6 +106,12 @@
       ]
     }
   },
+  "jest": {
+    "testPathIgnorePatterns": [
+      "/node_modules/",
+      "<rootDir>/tests/"
```
Comment (paninaro) [inline packages/gui/package.json:112]: Added because this is where the Playwright tests currently reside. We may want to move those somewhere else.

**Comment by ChiaMineJP** [comment]
Do you want to use both jest and playwright?

**Comment by paninaro** [comment]
> Do you want to use both jest and playwright?

Yes, @jfawChia is working on writing end-to-end tests using Playwright, while I'd like to use jest for unit tests. I'm open to other suggestions as well.

**Review by seeden — APPROVED** [review]
lgtm

**Pull request merged**